mods/overworld/underbrush.lua

```diff
@@ -287,6 +290,7 @@ minetest.register_node("overworld:natural_coconut", {
 		}
 	},
 	sounds = qtcore.node_sound_wood(),
+	on_flood = floodFunc,
```

Comment:
You want coconuts destroyed by water?

Reply:
it just made more sense than them making an air pocket in water